src/nvim/main.c

```diff
@@ -2070,3 +2072,29 @@ static void check_swap_exists_action(void)
     getout(1);
   handle_swap_exists(NULL);
 }
+
+
+// Check if $VIMRUNTIME points to a valid Neovim runtime path,
+// look for $VIMRUNTIME/autoload/provider/clipboard.vim,
+// which is a Neovim-specific file.
+// Print a warning if the file is not found.
+static void check_vimruntime(void)
+{
+  const char *runtime = os_getenv("VIMRUNTIME");
+  if (runtime) {
```

Comment:
This doesn't help the case where VIMRUNTIME env var is not defined, but runtime is invalid. E.g., if user forgot to install, or does not have permissions.

On one of my machines, I use:

    make CMAKE_EXTRA_FLAGS="-DCMAKE_INSTALL_PREFIX:PATH=~/bin/neovim" && make install

If `~/bin/neovim` gets deleted or something, the warning won't appear, but the user will still have the same questions that we are trying to address.